core/templates/dev/head/components/side_navigation_bar/SideNavigationBarDirective.js

```diff
@@ -26,7 +26,7 @@ oppia.directive('sideNavigationBar', [
         'side_navigation_bar_directive.html'),
       controller: ['$scope', '$timeout', function(
           $scope, $timeout) {
-        $scope.NAV_MODE = GLOBALS.NAV_MODE;
+        $scope.CURRENT_URL = window.location.pathname;
```

Comment:
Use angular version of `window.location.pathname` an since now it's not really a constant use camelCase
```suggestion
        $scope.currentUrl = $location.path();
```

Reply:
@vojtechjelinek after this change in SideNavigationBarDirective.js, the side nav bar has stopped working even after making equivalent changes in html. I also added '$location' to controller and function parameters. Still the expected behaviour is not established.

Reply:
FYI,
![image](https://user-images.githubusercontent.com/32777813/50245853-9c910300-03f9-11e9-9870-002d4232b353.png)
![image](https://user-images.githubusercontent.com/32777813/50245863-a31f7a80-03f9-11e9-9756-1ec2ac618fb5.png)

Reply:
I'm not sure, try to debug it, is there some error in browser console, try to print the value of `currentUrl` to check it contains correct value.

Reply:
@vojtechjelinek 
I did all the research possible, and fixed the issue.
![image](https://user-images.githubusercontent.com/32777813/50274425-3e990580-0463-11e9-8618-4219842b7eae.png)

But now, a new problem has arised. The $location api doesn't have the capability of reloading the entire page, unlike window.location(Look at the screenshot below). So the url gets changed on clicking a button on side nav bar, but it doesn't get reloaded. What should we do now? Use window.location itself?
![image](https://user-images.githubusercontent.com/32777813/50275288-05619500-0465-11e9-8105-c06af8e1ac29.png)

Reply:
Oh I see, we definitely shouldn't modify `$locationProvider`, so use `window.location.pathname` please.

Reply:
@vojtechjelinek Alright. Thanks.